static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -166,12 +171,19 @@ describe('groupEvents', function () {
   });
 
   describe('When the performance flag is enabled', () => {
-    it('renders new events table for performance', function () {
-      const org = initializeOrg({
+    let org;
+    let group;
+
+    beforeEach(() => {
+      org = initializeOrg({
         organization: {features: ['performance-issues-all-events-tab']},
       });
-      const group = TestStubs.Group();
+      group = TestStubs.Group();
+    });
```

Comment:
Any particular reason why you need to reinitialize the org and group before each test? I think it should be fine to just initialize them within the describe block